docs/proposals/Gallery/README.md

```diff
@@ -0,0 +1,74 @@
+# Component Name
+
+The `Gallery` component will accept an array of files and display them as
+thumbnails for images, other file types will instead display a generic icon
+corresponding to the file type. This would also have an option to show the
+images in a LightBox component on click of those images.
+
+## Accessibility
+
+We will support navigation through the items with tab and shift-tab. Since we
+cannot guarantee the `name` value passed in will be readable or useful, instead
+the alt value should reference the total number of files in the gallery and
+which file the user is currently focused on.
+
+eg. `alt=[1 of 4] attached files`
+
+Since the non-images will not be an actual thumbnail, rather some markup, we can
+use the `role=img` and `alt=...` attributes
+
+Triggering the onclick equivalent action should also be accessible via "space"
+and "return".
```

Comment:
Not sure what is being referred to in this line. Do you want this when a certain part of the component is focussed?  (Which part?) What under focus would trigger the click? (Space and return are probably too specific. I think we want whatever the browser keyboard action for a click would be.)